### DISCUSSION TITLE
Plot stationary transition plot as an effect of covariates

### DISCUSSION THREAD

**Discussion by Zehidul-EcoMove** [bmcclintock/momentuHMM#154] [Q&A]
Hi,

I have a 3-state model where the best model includes time of day (i.e., hour 0-23 hr) and the temperature with mixture (k=3). I tried to plot a customised graph of stationary state probability with each of the covariates at a time taking the other covariate as the mean. I am able to plot for the time of day having the values for est, se, uci and lci and time of day (0-23 hr) as given below. 

     est         se         lci       uci       cov
   0.2621786 0.02374462 0.2183584 0.3112905  0.00
   0.2648825 0.02399060 0.2205914 0.3144783  0.23
      

Similarly, I want to plot a stationary transition with respect to temperature keeping the time of day (hour = 18), however, I am unable to get the estimates for temperature with the following codes. 

###
plotStationary(mixmodel1_k3,
               plotCI = TRUE,
               alpha = 0.95,
               return = TRUE,
               covs = data.frame(temp = 32))

Regards
Zehidul

**Comment by bmcclintock** [comment]
I'm not sure I totally understand what you want to do, but, based on your description, I think you need to fix the `covs` argument:
```
plotStationary(mixmodel1_k3,
plotCI = TRUE,
alpha = 0.95,
return = TRUE,
covs = data.frame(hour = 18))
```
If this doesn't solve the issue, please provide a [minimal reproducible example](https://en.wikipedia.org/wiki/Minimal_reproducible_example#:~:text=In%20computing%2C%20a%20minimal%20reproducible,to%20be%20demonstrated%20and%20reproduced.).
